geonode/maps/migrations/24_initial.py

```diff
@@ -30,6 +30,7 @@ class Migration(migrations.Migration):
                 ('center_y', models.FloatField(verbose_name='center Y')),
                 ('last_modified', models.DateTimeField(auto_now_add=True)),
                 ('urlsuffix', models.CharField(max_length=255, verbose_name='Site URL', blank=True)),
+                ('renderer',  models.CharField(max_length=32, verbose_name='Renderer', blank=True)),
```

Comment:
@j-meyer Need to pull this out into its own migration file so that it can be applied to existing installs.